lib/series_arithmetic.rb

```diff
@@ -109,8 +109,10 @@ def rebase(year=nil)
       end
       new_base = self.at(Date.new(year)).to_f
     end
-    data.sort.each do |inner_date, value|
-      new_series_data[inner_date] = value / new_base * 100
```

Comment:
It occurs to me (esp since I don't really know what the code is doing) that in this place, if `new_base` can be nil or zero, we might rather that the code blow up than quietly return wrong results. If this is the case, please advise and I'll put it back the way it was.

Reply:
This is good. missing is the right value when we have a zero base.